pkg/impl/libraptorq/swig/swig.swigcxx

```diff
@@ -728,23 +728,27 @@ bool set_compression(Compress const compression);
 size_t local_cache_size (size_t const local_cache);
 size_t get_local_cache_size();
 
-namespace Impl {
-namespace Endian {
+} // namespace RaptorQ__v1
```

Comment:
is this file generated by swig?

Reply:
Again, nope.  This file is a SWIG source file – a (partial) interface definition of the wrapped C++ code to be converted into target (Go) language binding.